conf/spgwu.bess

```diff
@@ -44,6 +44,14 @@ except ValueError:
 except KeyError:
     print('measure value not set. Not installing Measure module.')
 
+# Detect mode. Default is dpdk
+mode = None
+try:
+    mode = conf["vdev"]
+except KeyError:
+    print('Selecting dpdk as default')
+    mode = 'dpdk'
+
```

Comment:
can we make the conf key to be mode itself? Also by default keep it None or unselected